internal/olm/operator/bundle/install.go

```diff
@@ -103,51 +102,54 @@ func (i *Install) setup(ctx context.Context) error {
 		return err
 	}
 
-	wd, err := os.Getwd()
-	if err != nil {
-		log.Error(err)
-	}
+	// FBC variables
+	directoryName := filepath.Join("/tmp", strings.Split(csv.Name, ".")[0]+"-index")
+	bundleChannel := strings.Split(labels[registrybundle.ChannelsLabel], ",")[0]
+	fileName := filepath.Join(directoryName, "testFBC")
 
-	// TODO (rashmi/venkat) might have to clean this up to remove/update some stuff.
 	f := &FBCContext{
-		BundleImage:       i.BundleImage,
-		Package:           labels[registrybundle.PackageLabel],
-		FBCDirContext:     "testdata",
-		FBCDirPath:        filepath.Join(wd, "testdata"),
-		FBCName:           "testFBC",
-		DescriptionReader: bytes.NewBufferString("foo"),
-		DefaultChannel:    "foo",
-		ChannelSchema:     "olm.channel",
-		ChannelName:       "foo",
+		BundleImage:    i.BundleImage,
+		Package:        labels[registrybundle.PackageLabel],
+		FBCDirName:     directoryName,
+		FBCName:        fileName,
+		DefaultChannel: bundleChannel,
+		ChannelSchema:  "olm.channel",
+		ChannelName:    bundleChannel,
 	}
 
 	// create entries for channel blob
 	entries := []declarativeconfig.ChannelEntry{
 		{
-			Name: "api-operator.v1.0.1",
+			Name: csv.Name,
```

Comment:
Let's make a note of this as well in our doc for further discussion and add a TODO item above this to capture this comment. When we render/extract our bundle image, there must be one way to get the channel entries instead of always assuming it would be just one entry.

Reply:
Cool, added it to the doc.